CODEOWNERS

```diff
@@ -1,3 +1,3 @@
-frontend/src/pages/faq* @stephaniehobson
-frontend/src/pages/premium* @stephaniehobson
-frontend/src/pages/index*
+frontend/src/pages/faq* @stephaniehobson @bcolsson
+frontend/src/pages/premium* @stephaniehobson @bcolsson
+frontend/src/pages/index* @stephaniehobson @bcolsson
```

Comment:
@bcolsson Did you want us to add you as a required reviewer to [frontend/pendingTranslations.ftl](https://github.com/mozilla/fx-private-relay/blob/main/frontend/pendingTranslations.ftl) as well?

Reply:
Tagging @peiying2 to weigh in on whether she'd want to be added. 

Peiying - if we do want to consder this, we could also use a team (like we do for other projects).

Reply:
Thank you Bryan for bringing this to my attention. 

@pdehaan  typically I conduct my reviews in the l10n repo, not here unless by special request. Bryan has been helping me to migrate Relay content to mozilla.org, thus his involvement here. He should be asked to review the PRs here. Thank you.